department-of-veterans-affairs/abd-vro
# MCP-2094 create end to end tests for mas

## What was the problem?

Sometimes the application would break but we had no way to detect it.

Associated tickets or Slack threads:

- [MCP-2094](https://amida.atlassian.net/browse/MCP-2094)

## How does this fix it?[^1]

These post-docker tests help catch some problems during CI.

However, since the endpoints run asynchronously, the test do not cover a lot of backend scenarios. The pre-docker integration tests cover some of that. I also have some ideas about calling endpoints that execute synchronously, but they need a bit more thought and planning.

## How to test this PR
- N/A


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
